common/src/main/scala/org/apache/celeborn/common/metrics/source/AbstractSource.scala

```diff
@@ -627,6 +627,8 @@ abstract class AbstractSource(conf: CelebornConf, role: String)
     sum
   }
 
+  // Do use this method to get metrics, because it will clear the timeMetrics queue
+  // Do not use the LogReporter to report metrics
```

Comment:
<img width="806" alt="image" src="https://github.com/user-attachments/assets/7bbb95e2-544f-4ce4-89d5-833e132105c1" />

I saw that, the `AbstractServlet`(Prometheus or Json) will call the `getMetricsSnapshot` and underlying `getMetrics` method.

Reply:
Does it mean that, if we enable LoggerSink, the prometheus can not retrieve timeMetrics then?

Reply:
Seems the loggerSink is a workaround to help cleanup the timeMetrics, if the `getMetrics` is not called for long time.